posthog/hogql/transforms/test/__snapshots__/test_lazy_tables.ambr

```diff
@@ -42,11 +40,9 @@
   
   SELECT cohort_people__new_person.id AS id 
   FROM (
-  SELECT cohortpeople.person_id AS person_id, cohortpeople.cohort_id AS cohort_id, cohortpeople.person_id AS cohort_people___person_id 
+  SELECT DISTINCT cohortpeople.person_id AS cohort_people___person_id, cohortpeople.person_id AS person_id, cohortpeople.cohort_id AS cohort_id 
   FROM cohortpeople 
-  WHERE equals(cohortpeople.team_id, 420) 
-  GROUP BY person_id, cohort_id, cohort_people___person_id 
-  HAVING ifNull(greater(sum(cohortpeople.sign), 0), 0)) AS cohort_people LEFT JOIN (
+  WHERE and(equals(cohortpeople.team_id, 420), false)) AS cohort_people LEFT JOIN (
```

Comment:
`false` - I'm assuming because wherever this test is hasn't setup the cohort version?

Reply:
Precisely. We have never stored any data in the cohort, so no need to query anything. Added an extra test to check it as well.